mssql-ballerina/Client.bal

```diff
@@ -0,0 +1,222 @@
+        if (self.clientActive) {
+            return nativeExecute(self, sqlQuery);
+        } else {
+            return error sql:ApplicationError("MsSQL Client is already closed, hence further operations are not allowed");
+        }
+    }
+
+    # Executes a batch of parameterized DDL or DML sql query provided by the user,
+    # and returns the summary of the execution.
+    #
+    # + sqlQueries - The DDL or DML query such as INSERT, DELETE, UPDATE, etc as `ParameterizedQuery` with an array
+    #                of values passed in
+    # + return - Summary of the executed SQL queries as `ExecutionResult[]` which includes details such as
+    #            `affectedRowCount` and `lastInsertId`. If one of the commands in the batch fails, this function
+    #            will return `BatchExecuteError`, however the JDBC driver may or may not continue to process the
+    #            remaining commands in the batch after a failure. The summary of the executed queries in case of error
+    #            can be accessed as `(<sql:BatchExecuteError> result).detail()?.executionResults`.
+    remote isolated function batchExecute(@untainted sql:ParameterizedQuery[] sqlQueries) returns sql:ExecutionResult[]|sql:Error {
+        if (sqlQueries.length() == 0) {
+            return error sql:ApplicationError(" Parameter 'sqlQueries' cannot be empty array");
+        }
+        if (self.clientActive) {
+            return nativeBatchExecute(self, sqlQueries);
+        } else {
+            return error sql:ApplicationError("MsSQL Client is already closed, hence further operations are not allowed");
+        }
+    }
+
+    # Executes a SQL stored procedure and returns the result as stream and execution summary.
+    #
+    # + sqlQuery - The query to execute the SQL stored procedure
+    # + rowTypes - The array of `typedesc` of the records that should be returned as a result. If this is not provided
+    #               the default column names of the query result set be used for the record attributes.
+    # + return - Summary of the execution is returned in `ProcedureCallResult` or `sql:Error`
+    remote isolated function call(@untainted string|sql:ParameterizedCallQuery sqlQuery, typedesc<record {}>[] rowTypes = [])
+    returns sql:ProcedureCallResult|sql:Error {
+        if (self.clientActive) {
+            return nativeCall(self, sqlQuery, rowTypes);
+        } else {
+            return error sql:ApplicationError("MsSQL Client is already closed, hence further operations are not allowed");
+        }
+    }
+
+    # Close the SQL client.
+    #
+    # + return - Possible error during closing the client
+    public isolated function close() returns sql:Error? {
+        self.clientActive = false;
+        return close(self);
+    }
+
+}
+
+# Client Configuration record for connection initialization
+#
+# + host - Hostname of the mssql server to be connected
+# + instanceName - Instance of mssql server to connect to on serverName
```

Comment:
field description is not clear

Reply:
Fixed with [`6ad0fb1`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/6ad0fb1ecfdca2bfc4d43985f5f4adcd7e6ddc37)